docs/dev/analysis/features/table/table-cell.rst

```diff
@@ -7,6 +7,20 @@ properties affecting its size, appearance, and how the content it contains is
 formatted.
 
 
+Cell.vAlign
+-----------
+
+Candidate protocol::
+
+    >>> from docx.enum.table import WD_VERTICAL_ALIGNMENT
+    >>> row = table.add_row()
+    >>> row.cells[0]
+    <docx.table._Cell object at 0x...>
+    >>> row.cells[0].vAlign
+    None
+    >>> row.cells[0].vAlign = WD_VERTICAL_ALIGNMENT.CENTER
+
```

Comment:
Let's add the alias `WD_ALIGN_VERTICAL` to the `WD_VERTICAL_ALIGNMENT` enumeration 
so folks can write e.g. `WD_ALIGN_VERTICAL.TOP` to correspond to `wdAlignVerticalTop`.
The capitalized snake-case version of the MS API name, as you have here, usually corresponds to the "prefix" of the member names. But in this case, the members all start with `wdAlignVertical..`.

There's a decorator defined for that, used like it is here:
https://github.com/python-openxml/python-docx/blob/master/docx/enum/dml.py#L46

Then you can use the WD_ALIGN_VERTICAL alias here in the import just as though it were a separate enumeration.

----

The property name should be `.vertical_alignment` rather than `.vAlign`. The library convention is to use snake case (no camel case) adapted from the MS API property name (`VerticalAlignment` in this case). We occasionally deviate from that but only when there's a compelling reason.